### PR TITLE
Add Hooks for Angle Coordinates to Roll

### PR DESCRIPTION
**Pull request by ChRen95**
Added more Hooks to roll and roll_pass.
Especially neutral_point, angle as well as entry angle to have this implemented never the less the plugin that one is using.

### CODE REVIEW

**Comment by ChRen95** [comment]
Afterwards we should consider a new release.

File: pyroll/core/roll_pass/hookimpls/roll.py [hunk 1]
```diff
@@ -33,11 +33,26 @@ def center(self: RollPass.Roll):
     return np.array([0, self.roll_pass.gap / 2 + self.nominal_radius])
 
 
+@RollPass.Roll.neutral_angle
+def neutral_angle(self: RollPass.Roll):
+    if self.has_value("neutral_point"):
+        return -np.arcsin(self.neutral_point / self.working_radius)
+    else:
+        return None
```
Comment (axtimhaus) [inline pyroll/core/roll_pass/hookimpls/roll.py]:
```suggestion
```

File: pyroll/core/roll_pass/hookimpls/roll.py [hunk 2]
```diff
@@ -33,11 +33,26 @@ def center(self: RollPass.Roll):
...
+
+
+@RollPass.Roll.neutral_point
+def neutral_point(self: RollPass.Roll):
+    if self.has_value("neutral_angle"):
+        return -np.sin(self.neutral_angle) * self.working_radius
+    else:
+        return None
```
Comment (axtimhaus) [inline pyroll/core/roll_pass/hookimpls/roll.py]:
```suggestion
```

File: pyroll/core/roll_pass/hookimpls/roll.py [hunk 3]
```diff
@@ -33,11 +33,26 @@ def center(self: RollPass.Roll):
...
+
+
 @RollPass.Roll.surface_velocity
 def surface_velocity(self: RollPass.Roll):
     if self.roll_pass.has_set("velocity"):
         if self.has_value("neutral_point"):
```
Comment (axtimhaus) [inline pyroll/core/roll_pass/hookimpls/roll.py:52]:
```suggestion
        if self.has_value("neutral_angle"):
```

File: pyroll/core/roll_pass/hookimpls/roll_pass.py [hunk 4]
```diff
@@ -140,10 +140,10 @@ def contact_area3(self: ThreeRollPass):
 @RollPass.velocity
 def velocity(self: RollPass):
     if self.roll.has_value("neutral_point"):
```
Comment (axtimhaus) [inline pyroll/core/roll_pass/hookimpls/roll_pass.py:142]:
```suggestion
    if self.roll.has_value("neutral_angle"):
```

File: pyroll/core/roll_pass/hookimpls/roll.py [hunk 5]
```diff
@@ -33,11 +33,26 @@ def center(self: RollPass.Roll):
...
+def neutral_angle(self: RollPass.Roll):
+    if self.has_value("neutral_point"):
+        return -np.arcsin(self.neutral_point / self.working_radius)
+    else:
+        return None
+
+
+@RollPass.Roll.neutral_point
+def neutral_point(self: RollPass.Roll):
+    if self.has_value("neutral_angle"):
```
Comment (axtimhaus) [inline pyroll/core/roll_pass/hookimpls/roll.py:44]:
```suggestion
    if self.has_set_or_cached("neutral_angle"):
```

**Review by axtimhaus — CHANGES_REQUESTED** [review]

**Comment by ChRen95** [comment]
Updated according to changes, ready for merging

**Review by axtimhaus — APPROVED** [review]

**Pull request merged**